tests/drivers/clock_control/stm32_clock_configuration/stm32u5_core/boards/pll_hse_fracn_160.overlay

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Warning: This overlay performs configuration from clean sheet.
+ * It is assumed that it is applied after clear_clocks.overlay file.
+ */
+
+/*
+ * Warning: HSE is not implemented on available boards, hence:
+ *          This configuration is only available for build
+ */
+
+&clk_hse {
+	status = "okay";
+	clock-frequency = <16777216>;
```

Comment:
IIUC, PLL frac doesn't have impact on PLL source but PLL output.

Reply:
correct, the PLL frac only has an impact on the PLL output. In this test case, I want to generate a 160 Mhz PLL output using this 16777216 Hz HSE value which requires the PLL to be in the fractional mode.